src/index.html

```diff
@@ -7,8 +7,9 @@
         <meta name="description">
 
         <link href="https://fonts.googleapis.com/css?family=Permanent+Marker" rel="stylesheet" type="text/css">
-        <script defer src="https://use.fontawesome.com/releases/v5.2.0/js/all.js"></script>
-        <link rel="stylesheet" href="/styles/main.css?v2.21.0">
+        <script defer src="https://use.fontawesome.com/releases/v5.0.6/js/all.js"></script>
```

Comment:
I'm guessing you need to get updated file for fontawesome and the main.css version number.